src/gui/mainwindow.cpp

```diff
@@ -282,14 +282,20 @@ void MainWindow::updateVersion(){
         if(kextstatOut.isEmpty())
             daemonWarning.append(tr("<br /><b>Warning:</b> System Extension by \"Fumihiko Takayama\" is not allowed in Security & Privacy. Please allow it and then unplug and replug your devices."));
 #elif defined(Q_OS_LINUX)
-            QProcess modprobe;
-            modprobe.start("modprobe", QStringList("uinput"));
-
-            if(!modprobe.waitForFinished())
-                qDebug() << "Modprobe error";
-
-            if(modprobe.exitCode())
-                daemonWarning.append(tr("<br /><b>Warning:</b> The uinput module could not be loaded. If this issue persists after rebooting, compile a kernel with CONFIG_INPUT_UINPUT=y."));
+            QFileInfo uinput("/dev/uinput");
+            if (!uinput.exists()){
+                QFileInfo uinput2("/dev/input/uinput");
+                if (!uinput2.exists()){
```

Comment:
This logic feels clumsy to me. I'm more inclined to have these be something like
```cpp
if (!QFileInfo("/dev/uinput").exists())
```
but ultimately @tatokis has the final word on things involving the GUI.